src/Castle.Windsor.Tests/Lifestyle/ScopedLifestyleExplicitAndMultipleThreadsTestCase.cs

```diff
@@ -102,6 +103,7 @@ public void Context_is_passed_onto_the_next_thread_TPL()
 				Assert.AreSame(instance, instanceFromOtherThread);
 			}
 		}
+#endif
```

Comment:
This failed for netcoreapp1.0. I am not sure why yet. I will investigate this a little more before we merge this PR. If you have any insights here I am open to suggestions. A cursory and glossy investigation is telling me this should pass. There is a gremlin here.

Reply:
Could you remove the conditional compilation so the pull request fails and I can see the error, or just paste the exception here.

Reply:
Here is the full failure: 

```
Failed   Context_is_passed_onto_the_next_thread_TPL
Error Message:
 System.AggregateException : One or more errors occurred. (  Expected: not equal to 6
  But was:  6
)
  ----> NUnit.Framework.AssertionException :   Expected: not equal to 6
  But was:  6

Stack Trace:
   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   at CastleTests.Lifestyle.ScopedLifestyleExplicitAndMultipleThreadsTestCase.Context_is_passed_onto_the_next_thread_TPL() in C:\code\Windsor\src\Castle.Windsor.Tests\Lifestyle\ScopedLifestyleExplicitAndMultipleThreadsTestCase.cs:line 102
--AssertionException
   at NUnit.Framework.Assert.ReportFailure(String message)
   at NUnit.Framework.Assert.AreNotEqual(Object expected, Object actual)
   at CastleTests.Lifestyle.ScopedLifestyleExplicitAndMultipleThreadsTestCase.<>c__DisplayClass1_0.<Context_is_passed_onto_the_next_thread_TPL>b__0() in C:\code\Windsor\src\Castle.Windsor.Tests\Lifestyle\ScopedLifestyleExplicitAndMultipleThreadsTestCase.cs:line 99
   at System.Threading.Tasks.Task.Execute()
```